src/ScriptCollection.cs

```diff
@@ -0,0 +1,20 @@
+namespace RunScript;
+
+using System.Collections.ObjectModel;
+using System.Text.Json.Serialization;
+
+using RunScript.Serialization;
+
+[JsonConverter(typeof(ScriptCollectionConverter))]
+public class ScriptCollection : KeyedCollection<string, ScriptMapping>
```

Comment:
I wonder if it might be worth setting the key comparer with the base constructor if you need OrdinalIgnoreCase or something?